markdown/events/2022/hackathon-october-2022.md

```diff
@@ -17,4 +17,54 @@ This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barc
 
 In addition to the in-person event, we will endeavour to run a hybrid option online in [gather.town](https://gather.town/).
 
-Please check back here soon - more details will be added to this page soon as we continue to plan the event.
+This hackathon is not meant to be a training event but aimed at people that already have a basic understanding of nextflow and/or nf-core. For anyone interested in learning nextflow, there are going to be workshops in the week before the hackathon. As soon as we have more information, we will let you know.
+
+<!--
+
+# Registration
+
+Registration will open soon. Please check back here and keep an eye out on twitter and Slack,
+but in the mean time - book the dates in your calendar!
+
+-->
+
+# Topic: Free-hacking!
+
+This time around we decided to not have an official topic. You are welcome to join the hackathon and work on your own pipelines or help improve existing ones. Of course you are also welcome to add to documentation!
```

Comment:
Topics was a bit of an unusual thing, so I don't think that we really need to point out that we're not doing that again.

```suggestion
# Groups
```